### DISCUSSION TITLE
Friendship status after following somebody

### DISCUSSION THREAD

**Discussion by emreond** [sbertix/Swiftagram#163] [Q&A] [answered]
Hello,

How can I know If user is followed or a request sent after I call `Endpoint.Friendship.follow(identifier)` .  On old library, friendShip status were returning but now object called `status` returning in success but It doesn't have any info about friendShip.

I am using version:  4.2.10

Thank You

**Comment by sbertix** [comment, marked as answer]
Hey @emreond, 

The value is still there, it's just not exposed. I'm gonna add it to `5.0.0`, though, and make sure the appropriate friendship is returned. 
For now, you can do this. 

```swift
let status: Status = /* assume this is the `Status` you get back from your request */
if let friendship = status.wrapper().friendshipStatus.optional().flatMap(Friendship.init) {
   /* do something with the friendship value */
}
```

**Reply by emreond** [reply]
Thank you so much.  Worked like a charm :)
